Scripts/Globals/PacketHandler.gd

```diff
@@ -39,7 +39,7 @@ func _read_packets(channel: int) -> void:
 	var packet_size: int = Steam.getAvailableP2PPacketSize(channel)
 	while packet_size > 0:
 		var packet: Dictionary = Steam.readP2PPacket(packet_size, channel)
-		open_packet(packet["steam_id_remote"], bytes2var(packet["data"]))
+		open_packet(packet["steamIDRemote"], bytes2var(packet["data"]))
```

Comment:
This GodotSteam commit changed it to `steam_id_remote`: https://github.com/GodotSteam/GodotSteam/commit/9497ecf3889d0769c24a4f3377d573439ebfd307

That commit is included on GodotSteam 3.10 and higher, so it looks like the version specified on the README is incorrect.

It's probably a better idea to try GodotSteam 3.10 or higher instead of that patch, and if it works, update the README: https://github.com/GodotSteam/GodotSteam/releases/tag/g333-s152-gs310